gallery/src/jsUtilPages/PolicyThreatLevelUtils/CategoryByPolicyThreatLevelExample.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { categoryByPolicyThreatLevel } from '@sonatype/react-shared-components';
+
+/* eslint-disable no-console */
+console.log('CategoryByPolicyThreatLevelExample');
+
+console.log(categoryByPolicyThreatLevel[5]); // 'severe'
```

Comment:
If we want to make this more visual we could adjust the example to just drop the output into the dom. Maybe a `nx-list` with the number next to the output of the function? (same with the other examples that just do a console log)

Reply:
> If we want to make this more visual we could adjust the example to just drop the output into the dom. Maybe a nx-list with the number next to the output of the function? (same with the other examples that just do a console log)

I considered that, but was worried it would result in more clutter than content.